app/controllers/users/notices_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Users::NoticesController < ApplicationController
+  def create
+    reciever = User.find(params[:reciever_id])
+    return render json: { status: "NOT NOTICE" } if reciever.is_noticed_in_30_minutes?
+
+    UnreadMailer.notice(current_user, reciever, params[:chat_url]).deliver_later!
```

Comment:
__[Misspell]__

"reciever" is a misspelling of "receiver"


<!-- Comment by [Sider](https://sider.review) -->